middlewares/validators/requests.ts

```diff
@@ -58,3 +60,36 @@ export const updateRequestsMiddleware = async (
     res.boom.badRequest(errorMessages);
   }
 };
+
+export const getRequestsMiddleware = async (req: OooRequestCreateRequest, res: OooRequestResponse, next: NextFunction) => {
```

Comment:
should we follow the same pattern like createRequest and updateRequest?

Reply:
which pattern?